### ISSUE TITLE
the auto configuration fails

### ISSUE THREAD

**Issue by itManuel** [fmdlc/ISP-Checker#1]
Hi, I'm trying this code, and found the part of the configuration (lines 26, 30 and 34 of the Makefile) fails to execute.

The workaround was logging in the graphana ui, change the admin passwd, and then change the passwd in the line to execute by hand.

Regards,

**Comment by fdelacruz-asapp** [comment]
@itManuel Could you please post the error message, by default `Kibana` admin must be `admin:admin` Are you using the images provided in the `DockerCompose` file?.

**Comment by itManuel** [comment]
Sure, the error is:
```shell
manolo@t1000:~/ISP-Checker/ISP-Checker$ make install
Make: Creating persistent Docker volumes
Make: Initializing datasources and dashboards
Make: Bootstrap done
influxdb-storage
grafana-storage
Creating influxdb ... done
Creating telegraf ... done
Creating grafana  ... done
make: *** [Makefile:24: install] Error 56
```
And then, I opened the web interface, was forced to change the password, and used the new password in the failed curl requests (I'm pasting one as example, but I did the 3 by hand):

```shell
manolo@t1000:~/ISP-Checker/ISP-Checker$ curl -s --location -X POST 'http://admin:superDuperPasswerd@localhost:3000/api/datasources' \
>   --header 'Content-Type: application/json' \
>   -d @./grafana/datasource.json
{"datasource":{"id":1,"orgId":1,"name":"InfluxDB","type":"influxdb","typeLogoUrl":"","access":"proxy","url":"http://influxdb:8086","password":"VerySecurePassword","user":"grafana","database":"telegraf","basicAuth":false,"basicAuthUser":"","basicAuthPassword":"","withCredentials":false,"isDefault":true,"jsonData":{},"secureJsonFields":{},"version":1,"readOnly":false},"id":1,"message":"Datasource added","name":"InfluxDB"}

manolo@t1000:~/ISP-Checker/ISP-Checker$ 
```

**Comment by fmdlc** [comment]
This bug must be fixed with version `v1.3` released today.
